src/main/java/com/stripe/model/Invoice.java

```diff
@@ -96,6 +95,10 @@
   @Setter(lombok.AccessLevel.NONE)
   ExpandableField<Charge> charge;
 
+  /** Time at which the object was created. Measured in seconds since the Unix epoch. */
+  @SerializedName("created")
+  Long created;
```

Comment:
Why does this look added even though it's already in master per https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Invoice.java#L32 ?

Reply:
Yup it is comparing with the autogen branch, not the master